src/Components/LandingPage/LandingPage.scss

```diff
@@ -25,16 +25,16 @@
   margin-top: 15%;
   perspective: 1000px;
   perspective-origin: 50% 100px;
-  position: absolute; 
+  position: absolute;
```

Comment:
Should include main-div in the media-query.  The box moves up into the title at smaller screen sizes. 

Shows at a screen width of less 1096 pixels